packages/react/src/auto/mui/inputs/MUIAutoJSONInput.tsx

```diff
@@ -11,24 +11,23 @@ export const MUIAutoJSONInput = (
     control?: Control<any>;
   } & Partial<TextFieldProps>
```

Comment:
`Partial<Omit<TextFieldProps, "onChange">>` since passing in `onChange` will now do nothing